src/doc-editors/ACLEditor.vue

```diff
@@ -230,12 +230,18 @@ export default defineComponent({
         if (this.localDoc.tags && this.localDoc.tags.length > 0) {
           return this.localDoc.tags.join(' ')
         }
+        this.$emit('tags-invalid', true)
         return ''
       },
       set: function(tags: string): void {
         this.localDoc.tags = tags.length > 0 ? _.map(tags.split(' '), (tag) => {
           return tag.trim()
         }) : []
+        if (tags.trim() == '' || tags.length < 3) {
```

Comment:
```suggestion
        if (tags.trim() === '' || tags.length < 3) {
```

Strict equality operators (triple `=` as opposed to double `=`) are considered best practice, let's try to use them whenever possible